README.md

```diff
@@ -73,15 +73,15 @@ require(
         ],
         captions: [{
             url: 'https://www.somelovelycaptionsurl.com/captions/$segment$', // $segment$ required for replacement for live
-            segmentLength: 3.84 // Required to calculate live subtitle segment to fetch & live subtitle URL.
-            cdn: 'cdn1' // Displayed by Debug Tool
+            segmentLength: 3.84, // Required to calculate live subtitle segment to fetch & live subtitle URL.
+            cdn: 'cdn1', // Displayed by Debug Tool
           }, {
             url: 'https://www.somelovelycaptionsurl2.com/captions/$segment$',
-            segmentLength: 3.84 
-            cdn: 'cdn1' 
+            segmentLength: 3.84,
+            cdn: 'cdn1',
```

Comment:
Just noticed @RelentlessExploration that there's trailing commas in here. Although not illegal from a ES5 point of view, we do have an eslint rule to not allow them so for consistency with our codebase it would be great if you could remove any trailling commas 😄 Thanks

Reply:
Will do, thanks!